app/src/test/java/io/intrepid/contest/screens/contestoverview/ContestOverviewPresenterTest.java

```diff
@@ -58,6 +58,36 @@ public void onViewCreatedShouldShowContestNameWhenSuccessfullyRetrievedContestDe
     }
 
     @Test
+    public void onViewCreatedShouldShowRatingGuide() throws Exception {
```

Comment:
I think it should just show the Rating guide on that screen regardless. Maybe I misunderstood this part of the screen, but since this is just a static rubric or criteria, I think it is just a static part of our view. does that make sense?